joomla/joomla-cms
# [4.0] Smart Search: Adding debug information

### Summary of Changes
When running the indexer of Smart Search in the backend, the output is very unsatisfying for debugging. When debugging is enabled, this PR adds a list of all contexts that are being indexed with the total number of items to index and the currently indexed items.

### Testing Instructions
1. Apply PR
2. Run ```npm i```
3. Enable debugging in global configuration
4. Run the indexer of Smart Search in the backend and see the
